.golangci.yml

```diff
@@ -33,12 +33,14 @@ linters:
     - gofumpt           # prefer standard gofmt
     - golint            # deprecated by Go team
     - gomnd             # some unnamed constants are okay
+    - ifshort           # deprecated by author
     - interfacer        # deprecated by author
     - ireturn           # "accept interfaces, return structs" isn't ironclad
     - lll               # don't want hard limits for line length
     - maintidx          # covered by gocyclo
     - maligned          # readability trumps efficient struct packing
     - nlreturn          # generous whitespace violates house style
+    - nosnakecase       # deprecated in https://github.com/golangci/golangci-lint/pull/3065
```

Comment:
This will be deprecated in next golangci-lint but disabled for now because it causes lint errors on imported generated code: https://github.com/sivchari/nosnakecase/issues/16